docs/resources/Guild.md

```diff
@@ -521,6 +521,106 @@ A partial [guild](#DOCS_RESOURCES_GUILD/guild-object) object. Represents an Offl
+| enabled                 | boolean                                                                                                                 | is guild onboarding enabled?                                   |
+
+
+###### Onboarding Prompt Structure
+
+| Field            | Type                                                                                                    | Description                                            |
+| ---------------- | ------------------------------------------------------------------------------------------------------- | ------------------------------------------------------ |
+| id               | snowflake                                                                                               | the id of the prompt                                   |
+| options          | array of [prompt option](#DOCS_RESOURCES_GUILD/guild-onboarding-object-prompt-option-structure) objects | the options available to the prompt                    |
+| title            | string                                                                                                  | the title of the prompt                                |
+| single_select    | boolean                                                                                                 | can only one option of the prompt be selected at time? |
+| required         | boolean                                                                                                 | is this prompt required in onboarding flow?            |
+| in_onboarding    | boolean                                                                                                 | is this prompt in the onboarding flow?                 |
+| type             | [prompt type](#DOCS_RESOURCES_GUILD/guild-onboarding-object-prompt-type)                                | the type of the prompt                                 |
+
+
+###### Prompt Option Structure
+
+| Field       | Type                | Description                                                                               |
+| ----------- | ------------------- | ----------------------------------------------------------------------------------------- |
+| id          | snowflake           | the id of the option                                                                      |
+| channel_ids | array of snowflakes | the channels opted into when this option is selected                                      |
+| role_ids    | array of snowflakes | the roles assigned when this option is selected                                           |
+| emoji_id    | ?snowflake          | the [emoji id](#DOCS_REFERENCE/image-formatting), if the emoji is custom                  |
+| emoji_name  | ?string             | the emoji name if custom, the unicode character if standard, or `null` if no emoji is set |
```

Comment:
In light of #5393, is there a chance this can be changed to the standard remoji representation also used in other APIs (as listed in that discussion)?

Reply:
ya, it looks like we can just do this without any backwards incompat. Going to make this change and update the example to match the docs. Please note that old versions of onboarding may still have this bug present and I'm not going to go back and fix those :)

Reply:
Just to be clear, I'm talking about using the `emoji: { name, id, animated }` structure here, not (just) about `emoji_name` being null - but rather, to use that partial emoji structure instead of those two `emoji_name`/`emoji_id` keys. Or probably use both representations for client backwards compat. Because otherwise, 5+ endpoints use one format and 3 endpoints use another format, which makes the API even more inconsistent with itself.